.changeset/wet-items-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/backend-cli': patch
+---
+
+`amplify sandbox --version` returns version of the sandbox package
```

Comment:
Seems like this message needs to be updated

Reply:
Good catch!

Reply:
Done 24038c422